mw-webapp/src/component/editableTextarea/editableTextarea.module.scss

```diff
@@ -1,7 +1,38 @@
 @import "src/styles/variables";
 
-$editableTextarea-width: 100%;
```

Comment:
Why did you deleted this variable? It is using below  in this file

Reply:
I moved this variable to the _variables.scss file.